packages/stores/src/ui-config/manage-liquidity/add-concentrated-liquidity.ts

```diff
@@ -205,8 +205,7 @@ export class ObservableAddConcentratedLiquidityConfig {
 
   /** Used to ensure ticks are cleanly divisible by. */
   protected get tickDivisor() {
-    // TODO: use tickspacing from pool going forward
-    return new Int(1000);
+    return new Int(this.pool?.tickSpacing ?? 100);
```

Comment:
Why can't we always use the pool's tick spacing? What is the purpose of harcoding 100? Is it because we might not have the pool data available in some edge cases yet?

Reply:
Yeah the pool data needs to load